libethcore/EVMSchedule.h

```diff
@@ -146,10 +147,26 @@ static const EVMSchedule ConstantinopleFixSchedule = [] {
     return schedule;
 }();
 
+static const EVMSchedule IstanbulSchedule = [] {
+    EVMSchedule schedule = ConstantinopleFixSchedule;
+    schedule.version = 1;
+    return schedule;
+}();
+
 static const EVMSchedule ExperimentalSchedule = [] {
-    EVMSchedule schedule = ConstantinopleSchedule;
+    EVMSchedule schedule = IstanbulSchedule;
     schedule.blockhashGas = 800;
     return schedule;
 }();
+
+inline EVMSchedule const& evmScheduleForAccountVersion(u256 const& _version)
+{
+    if (_version == 0)
+        return ConstantinopleFixSchedule;
+    else if (_version == IstanbulSchedule.version)
+        return IstanbulSchedule;
+    else
+        return DefaultSchedule;
```

Comment:
Does the EIP specifies the rules here?

Reply:
No, but this should never happen, as all new accounts are created either with version 0 or with the version of the current fork.
We could add an assertion here.

Reply:
Go with assert.